### DISCUSSION TITLE
Downtime for sx.fukt.lol

### DISCUSSION THREAD

**Discussion by iotalol** [searxng/searx-instances#267] [Report instance downtime]
I am having an issue with SearXNG after updating Docker, I am getting "Too many Requests" looking for ways how to fix this.

**Comment by iotalol** [comment]
It seems that this is an issue on the reverse proxy.. Weird

**Comment by iotalol** [comment]
Fixed at 1:55 PM PST
Issue was reverse proxy 
Thanks!

**Comment by unixfox** [comment]
Thank you for the notice.
